### PR TITLE
Solve windows must run as admin issue

### PR DESCRIPTION
**Pull request by somdoron**

### CODE REVIEW

File: app/ZenNode.js [hunk 1]
```diff
@@ -157,10 +159,17 @@ class ZenNode {
...
     }
-    if (process.env.ZEN_NODE_API_PORT) {
-      args.push('--api', `127.0.0.1:${process.env.ZEN_NODE_API_PORT}`)
+    if (isWindows()) {
+      args.push('--api', `localhost:${getPort()}`)
     }
+    if (process.env.ZEN_NODE_API_PORT) {
 
+      if (isWindows()) {
+        args.push('--api', `localhost:${process.env.ZEN_NODE_API_PORT}`)
```
Comment (goldylucks) [inline app/ZenNode.js]: if user is on windows and `process.env.ZEN_NODE_API_PORT` is defined, we'll be passing two `--api` flags to the node, is that okay? "lest flag wins"?

**Comment by somdoron** [comment]
Yes

File: app/ZenNode.js (hunk 1 above)
Comment (somdoron) [reply app/ZenNode.js]: it is fine. I also handle that case in use localhost instead of 127.0.0.1

Comment (somdoron) [reply app/ZenNode.js]: Sorry, we don't actually want that. I will fix

**Comment by somdoron** [comment]
@goldylucks the issue is fixed. Please review again

**Comment by goldylucks** [comment]
LGTM. Haven't QA'd tho :)

**Pull request merged**